src/segment-parser.js

```diff
@@ -247,7 +257,28 @@
       // used to specify some forms of timing and control data that we
       // do not currently use.
       if (afflag > 0x01) {
-        offset += data[offset] + 1;
+        adaptation_field_length = data[offset];
+
+        if (adaptation_field_length > 0) {
+          afftemp = data[offset + 1];
+
+          discontinuity_indicator = (afftemp & 0x80) >>> 7;
+          random_access_indicator = (afftemp & 0x40) >>> 6;
+          elementary_stream_priority_indicator = (afftemp & 0x20) >>> 5;
+          PCR_flag  = (afftemp & 0x10) >>> 4;
+          OPCR_flag = (afftemp & 0x08) >>> 3;
+          splicing_point_flag = (afftemp & 0x04) >>> 2;
+          transport_private_data_flag = (afftemp & 0x02) >>> 1;
+          adaptation_field_extension_flag  = afftemp & 0x01;
```

Comment:
Purely for completeness. I wanted to parse it out to compare it with other output but we don't really need them here yet (maybe ever) but at least now they'll be in the history.